src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/SdkContentTests.cs

```diff
@@ -211,3 +215,21 @@ private static string RemoveDiffMarkers(string source)
         return diffSegmentRegex.Replace(result, "@@ ------------ @@");
     }
 }
+
+public class SdkTestsExclusionsHelperFixture : IDisposable
```

Comment:
This fixture class seems unneeded. Since the configuration here is specific to each test case, why not just have each test case instantiate the helper, just as was done for the license scan test.